cogs/utils/ctftimeapiutils.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+import config
```

Comment:
Lets just set a header as a constant. Don't think this needs to go in the config (I'm also trying to remove the config file currently)
```suggestion
CTFTIME_HEADER = "Whatever-you-were-using-in-the-config"
```